junit-common/src/main/java/org/jboss/weld/junit/MockInterceptor.java

```diff
@@ -220,6 +231,17 @@ public Builder type(InterceptionType type) {
             return this;
         }
 
+        /**
+         * TODO
```

Comment:
Something like:
`Allows to specify a bean class of an interceptor. This is only required if you later on need to enforce interceptor ordering via Weld.enableInterceptors(). Note that such ordering corresponds to enabling interceptors via beans.xml (e.g. per bean archive).`